index.bs

```diff
@@ -126,7 +128,9 @@ for example by making the entry be a part of the PermissionDescriptor.
 
 <div algorithm>
 An [=/entry=] |entry| has associated <dfn for=entry>query permission steps</dfn>,
-which take a {{FileSystemHandlePermissionDescriptor}} and return a {{PermissionState}}.
+which take a {{FileSystemHandlePermissionDescriptor}} |descriptor|
+and an optional [=environment settings object=] |settings|,
```

Comment:
Why optional? It seems this is passed in every invocation.

(here and the other "XXX permission steps", and therefore the use of "current settings object" if not passed)

Reply:
Optional because similar algorithms in the permissions spec had it optional. But agreed that there is no reason to have it optional here, so changed that.